pandas/errors/__init__.py

```diff
@@ -202,6 +202,25 @@ class NumbaUtilError(Exception):
     """
 
 
+class DuplicateLabelError(ValueError):
```

Comment:
Any reason to inherit ValueError instead of just Exception? The former seems a little strange to me

Reply:
I vaguely recall a Python style guide / book recommending that libraries not define custom exceptions from scratch, and instead inherit from the sensible concrete exception class in the standard library. I can't find the reference right now though.